src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTokenExtensions.vb

```diff
@@ -57,6 +57,31 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         <Extension()>
+        Public Function IsGlobalStatementContext(token As SyntaxToken, position As Integer) As Boolean
```

Comment:
Is `position` assumed to be in the full-span of `token`?  Should that be enforced (or, at least, asserted)?